sdk/src/main/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayData.java

```diff
@@ -267,7 +267,7 @@ public String getLabel() {
      * <p>If no URL was specified, this will return {@code null}.
      */
     @Nullable
-    public String getUrl() {
+    public String getLinkUrl() {
```

Comment:
May be worth using an explicit `@JsonGetter("link_url")` on this and the other getters to ensure stability of the JSON names (and make it clear that these getters are being used by Jackson).